Dockerfile

```diff
@@ -11,3 +11,27 @@ RUN curl -sSf https://static.rust-lang.org/rustup.sh | sh
 ADD package_installs.R /tmp/package_installs.R
 RUN Rscript /tmp/package_installs.R && \
     bash -c "rm -Rf /tmp/Rtmp*"
+
+# Miniconda (used by time-series API)
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+RUN apt-get update --fix-missing && \
+    apt-get install -y wget bzip2 ca-certificates curl git && \
+    apt-get clean && \
```

Comment:
It may be better to reuse the `clean-layer.sh` we use in other Dockerfiles https://github.com/Kaggle/docker-rstats/blob/master/clean-layer.sh